lib/irb/command/copy.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module IRB
+  module Command
+    class Copy < Base
+      category "Workspace"
+      description "Copy command output to clipboard"
+
+      help_message(<<~HELP)
+        Usage: copy (command)
+      HELP
+
+      def execute(arg)
+        value = irb_context.workspace.binding.eval(arg)
```

Comment:
When this command is executed without argument, `arg` will be an empty string, and the value `nil` will be copied.
I think we can show a usage message for this command, or just copy the previous result just like executing `copy _`.

Reply:
Good catch, I updated it to do the latter.